kasa/iot/iottimezone.py

```diff
@@ -0,0 +1,156 @@
+"""Module for io device timezone lookups."""
+
+from __future__ import annotations
+
+import logging
+from datetime import datetime, tzinfo
+
+from zoneinfo import ZoneInfo
+
+_LOGGER = logging.getLogger(__name__)
+
+
+def get_timezone(index: int) -> tzinfo:
+    """Get the timezone from the index."""
+    if index > 109:
+        _LOGGER.error(
+            "Unexpected index %s not configured as a timezone, defaulting to UTC", index
+        )
+        return ZoneInfo("Etc/UTC")
+
+    name = TIMEZONE_INDEX[index]
+    return ZoneInfo(name)
+
+
+def get_timezone_index(name: str) -> int:
```

Comment:
Do we really need this for something?

Reply:
We can enable setting the timezone with this in a subsequent PR

Reply:
Hmm, could we just reverse the index<->timezone mapping for that purpose?

Reply:
This will allow setting a time zone to any valid IANA time zone the device supports. The reverse mapping only contains 109 time zones but there are many more. I.e. Europe/London instead of GB etc.